.travis.yml

```diff
@@ -11,18 +9,15 @@ php:
     - 5.5
     - 5.6
     - 7.0
-    - hhvm
```

Comment:
@stof , i tried it here : https://travis-ci.org/akeneo-labs/spreadsheet-parser/builds/118362560